src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -10727,8 +10727,14 @@ public void LoadingAnalyzerNetStandard13()
             var srcFile = analyzerDir.CreateFile(srcFileName).WriteAllText("public class C { }");
 
             var result = ProcessUtilities.Run(s_CSharpCompilerExecutable, arguments: $"/nologo /t:library /analyzer:{analyzerFileName} {srcFileName}", workingDirectory: analyzerDir.Path);
+            var outputWithoutPaths = Regex.Replace(result.Output, " in .*", "");
             AssertEx.AssertEqualToleratingWhitespaceDifferences(
-                $"warning AD0001: Analyzer 'TestAnalyzer' threw an exception of type 'System.NotImplementedException' with message '28'.", result.Output);
+                $@"warning AD0001: Analyzer 'TestAnalyzer' threw an exception of type 'System.NotImplementedException' with message '28'.
+System.NotImplementedException: 28
+   at TestAnalyzer.get_SupportedDiagnostics()
+   at Microsoft.CodeAnalysis.Diagnostics.AnalyzerManager.AnalyzerExecutionContext.<>c__DisplayClass20_0.<ComputeDiagnosticDescriptors>b__0(Object _)
```

Comment:
> at Microsoft.CodeAnalysis.Diagnostics.AnalyzerManager.AnalyzerExecutionContext.c__DisplayClass20_0.<ComputeDiagnosticDescriptors>b__0(Object _)

This looks very brittle. Can we instead just verifying containment of specific parts of the text?

Reply:
Not easily, but it's rare for this portion of the tests to need changes so I'm not especially concerned at this point. (Most tests only care that AD0001 is _not_ reported.)